lib/src/signed_video_internal.h

```diff
@@ -175,6 +173,7 @@ struct _signed_video_t {
   size_t arbitrary_data_size;  // Size of |arbitrary_data|.
 
   bool has_public_key;  // State to indicate if public key is received/added
+  bool sv_test_on;  // State to indicate if public key is received/added
```

Comment:
Move this to the bottom of this struct well separated from the other members and add  like a section for test and debug variables.